make_queue/forms.py

```diff
@@ -153,3 +154,32 @@ class FreeSlotForm(forms.Form):
         initial=1, label=_("Machine type"))
     hours = IntegerField(min_value=0, initial=1, label=_("Duration in hours"))
     minutes = IntegerField(min_value=0, max_value=59, initial=0, label=_("Duration in minutes"))
+
+
+class BaseMachineForm(forms.ModelForm):
+    class Meta:
+        model = Machine
+        fields = "__all__"
+        widgets = {
+            "status": SemanticChoiceInput(),
+            "machine_type": SemanticChoiceInput(),
+            "location": MazemapSearchInput(url_field="location_url"),
+        }
+
+
+class CreateMachineForm(BaseMachineForm):
+    class Meta(BaseMachineForm.Meta):
+        exclude = ["status"]
```

Comment:
Why would we need to exclude status on creation? I think that it would be more useful to include it, as sometimes (e.g. for the printers) we would like to set it to `Maintenance` or `Out of order` at creation. For sewing machines on the other hand, it would be okay to set it to `Free` at creation.

Reply:
I thought that you wouldn't publish a machine on the website until it was ready. I see the use case now, so can add the field back again.